pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -136,6 +136,13 @@ product:
   clusterManagement: Cluster Management
   monitoring: Monitoring
   mcapps: Multi-cluster Apps
+  version: Version
+  versionChecking: (checking...)
+  networkStatus: Network status
+  kubernetesVersion: Kubernetes
+  containerEngine: CE
```

Comment:
I'm fine with using "CE" in the status bar to save space, but we never specify what this abbreviation means anywhere in Rancher Desktop. Perhaps, we can specify "Container Engine: moby" or "Container Engine(CE): Moby" in the popover? This will provide people a quick way to see what CE stands for if it's not immediately obvious from the outset.

![image](https://github.com/rancher-sandbox/rancher-desktop/assets/835961/6b59ac00-4dc8-4879-add3-34e0722fed25)

Reply:
I think `Container Engine: moby` is fine; no need to repeat the `CE` abbreviation in the hover tip.